dask/dataframe/tests/test_groupby.py

```diff
@@ -2148,6 +2148,123 @@ def test_groupby_shift_with_freq():
     assert_eq(df_result, ddf.groupby("b").shift(periods=-2, freq="D", meta=df_result))
 
 
+@pytest.mark.parametrize("npartitions", [1, 3])
+@pytest.mark.parametrize("method", ["average", "min", "max", "first", "dense"])
+@pytest.mark.parametrize("ascending", [True, False])
+@pytest.mark.parametrize("na_option", ["keep", "top", "bottom"])
+@pytest.mark.parametrize("pct", [False, True])
+@pytest.mark.parametrize("axis", [0, 1])
```

Comment:
This is fairly large number of combinations, which can be pared down if desired to reduce test time. It takes about 45 seconds on my local machine.

Reply:
Yeah this does feel like overkill. When culling these down it might be helpful to remember that we don't need to be testing pandas functionality here, so we probably don't need to test every option for every kwarg.

Reply:
Makes sense, thanks for the guidance

Reply:
Pared down tests now take about 5 seconds on my local machine